src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -80,7 +80,12 @@ QgsPostgresProvider::pkType( const QgsField &f ) const
       // (in which case we could use pktUint64)
       // we'll have to use a Map type.
```

Comment:
This comment is superseded?

Reply:
It is, I've removed the part that mentions the bug and the treatment as pktUint64.

Reply:
Done, see 9607570